JS Basics/Objects and Array methods/Problem_10/index.js

```diff
@@ -0,0 +1,20 @@
+function youngestMale(arr) {
+    let youngestMale = arr.find((person) => person.gender === "male")
+    arr.forEach((person) => {
+        if(person.gender === "male" && person.age < youngestMale.age) {
+            youngestMale = person;
+        }
+    });
+    return youngestMale.firstname + " " + youngestMale.lastname;
+}
```

Comment:
Use array.filter to get the array of males only -> then iterate through it and find the youngest male.